docs/source/command_line.rst

```diff
@@ -576,6 +611,9 @@ in developing or debugging mypy internals.
     If set, this flag will display a full traceback when mypy
     encounters a fatal error.
 
+``--raise-exceptions``
+    Raise exception on fatal error.
+
 ``--custom-typing MODULE``
```

Comment:
Since you changed the flag, this should be `--custom-typing-module`.

Reply:
Oh, you're right! Thanks!